101-azurefirewall-create/azuredeploy.parameters.json

```diff
@@ -3,25 +3,10 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "virtualNetworkName": {
-      "value": "yourvnetname"
+      "value": "GEN-UNIQUE"
     },
-    "vnetAddressPrefix": {
-      "value": "10.0.0.0/16"
-    },
-    "subnetName": {
-      "value": "yoursubnet"
-    },
-    "subnetPrefix": {
-      "value": "10.0.0.0/24"
-    },
-    "firewallName": {
-      "value": "yourappgateway"
-    },
-    "azureFirewallSubnetAddressPrefix": {
-      "value": "10.0.1.0/24"
-    },
-    "publicIPName": {
-      "value": "yourPublicIPName"
+    "location": {
+      "value": "West CentralUS"
```

Comment:
this isn't a valid location, but since CI passed in westus seems you can remove this?